frontend/src/components/forms/Slider.tsx

```diff
@@ -0,0 +1,42 @@
+import { withStyles } from '@material-ui/core/styles';
+import { Slider as MaterialSlider } from '@material-ui/core';
+
+export const Slider = withStyles({
+  root: {
+    color: '#0ec679',
```

Comment:
```suggestion
const COLOR_EMERALD = '#0ec679';

export const Slider = withStyles({
  root: {
    color: COLOR_EMERALD,
```

Voisi nimetä nämä värit, niin on sitten helpompi yhdistää `shared.scss`:n vastaavaan väriin.

Scss muuttujia ei taida pystyä suoraan käyttämään

Reply:
Jos haluaa koittaa saada scss muuttujat suoraan, niin tämä voisi toimia, mutta siinä täytyisi käydä vähän configeja muuttamassa ja exportata `shared.scss` tiedostosta muuttujat:
https://til.hashrocket.com/posts/sxbrscjuqu-share-scss-variables-with-javascript

Reply:
Ilmeisesti jos scss-filun nimeää moduulina niin voi importtaa muuttujia ilman mitään ekstraa. Laitoin värit omaan tiedostoon (sivun tyylit räjähti jostain syystä kun muutin shared.scss -> shared.module.scss)